cluster/local/integration_tests.sh

```diff
@@ -58,16 +58,26 @@ if [ "$skipcleanup" != true ]; then
     trap cleanup EXIT
 fi
 
-echo_step "creating k8s cluster using kind"
-"${KIND}" create cluster --name="${K8S_CLUSTER}"
+readonly DEFAULT_KIND_CONFIG="kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+"
+if $skipcleanup && $(echo $("${KIND}" get clusters) | grep -wq "${K8S_CLUSTER}"); then
```

Comment:
lot of unnecessary code added just to support to output:
```
"existing kind cluster: <somecluster>
```
The script already has clear error text:
```
>>>>>>> creating k8s cluster using kind
ERROR: failed to create cluster: node(s) already exist for a cluster with the name "build-01fa6cad-inttests"
```
It could be useful only if you have to run integration tests multiple times during **one** pipeline.